src/main/kotlin/com/bugsnag/android/gradle/BugsnagPlugin.kt

```diff
@@ -497,6 +498,16 @@ class BugsnagPlugin : Plugin<Project> {
         }
     }
 
+    internal fun getSourcemapUploadEndpoint(bugsnag: BugsnagPluginExtension): String {
+        return when (val endpoint = bugsnag.endpoint.get()) {
+            UPLOAD_ENDPOINT_DEFAULT -> endpoint
```

Comment:
This looks like it passes through the endpoint (host only) if it is set, or left as the default value?

`@bugsnag/source-maps` needs the absolute URL to be passed (including the path). So you can either construct the full path, or don't pass any value and it will use the default. As it currently stands this will try to send a react native source map payload to the root of the upload server and will fail, because the payloads are different.

(Note that the path should be `react-native-source-map` per my other comment.

Reply:
Updated so that the default endpoint is not passed as part of the command shelled out. It still remains as a property on the task as this is important for Gradle's task caching.